cli/CMakeLists.txt

```diff
@@ -30,6 +30,12 @@ if(LIBNUMA_FOUND)
   set(QUICKSTEP_HAVE_LIBNUMA TRUE)
 endif()
 
+if (BUILD_SHARED_LIBS)
+  set(GFLAGS_LIB_NAME gflags_nothreads-shared)
+else()
+  set(GFLAGS_LIB_NAME gflags_nothreads-static)
+endif()
```

Comment:
FYI, I've used both `gflags` and `glog` as git submodules for a private repo, with the condition `NOT TARGET gflags_nothreads-static` to choose whether compile `gflags`.